kernel/aster-nix/src/process/signal/sig_queues.rs

```diff
@@ -142,6 +142,26 @@ impl SigQueues {
         let idx = (signum.as_u8() - MIN_RT_SIG_NUM) as usize;
         &mut self.rt_queues[idx]
     }
+    
+    pub fn sig_pending(&mut self) -> SigSet {
```

Comment:
`sig_pending` can accept `&self` as parameter, instead of `&mut self`

Reply:
Thank you for your patient review, I learned a lot ~~  FIXED！